### PR TITLE
[jansson] new port required by nghttp2 #1122

### PR DESCRIPTION
**Pull request by atkawa7**

### CODE REVIEW

File: ports/jansson/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  set(JANSSON_STATIC_CRT  OFF)
+endif()
+
+
+vcpkg_configure_cmake(
+    SOURCE_PATH ${SOURCE_PATH}
+	OPTIONS
+	-DJANSSON_STATIC_CRT=${JANSSON_STATIC_CRT}
+  -DJANSSON_EXAMPLES=OFF
+  -DJANSSON_WITHOUT_TESTS=OFF
```
Comment (ras0219-msft) [inline ports/jansson/portfile.cmake]: Should this be ON? (implying that tests would be _disabled_)

File: ports/jansson/portfile.cmake [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  -DJANSSON_BUILD_SHARED_LIBS=ON
```
Comment (ras0219-msft) [inline ports/jansson/portfile.cmake]: Should this be triggered off of `VCPKG_LIBRARY_LINKAGE`?

File: ports/jansson/portfile.cmake [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO akheron/jansson
+    REF v2.10
+    SHA512  0859c8a29c1cb7e406d3eb369285500e98ee23677e3a6af6cd81511c57a6aa8dbabe4a8c7b673a3000203b68f74d84786817aec76cb1e85af217268159bda4b6
+    HEAD_REF master
+)
+
+if(VCPKG_LIBRARY_LINKAGE STREQUAL "static")
```
Comment (ras0219-msft) [inline ports/jansson/portfile.cmake]: This should be `VCPKG_CRT_LINKAGE`

Comment (atkawa7) [inline ports/jansson/portfile.cmake]: @ras0219-msft Thanks. Yes it should be

File: ports/jansson/portfile.cmake (hunk 2 above)
Comment (atkawa7) [inline ports/jansson/portfile.cmake]: @ras0219-msft  Yep. Gonna make some changes

File: ports/jansson/portfile.cmake (hunk 1 above)
Comment (atkawa7) [inline ports/jansson/portfile.cmake]: @ras0219-msft Definitely

**Comment by ras0219-msft** [comment]
Thanks for fixing those issues!

**Pull request merged**